### DISCUSSION TITLE
Is there a way to set `given name` or `family name` as optional?

### DISCUSSION THREAD

**Discussion by Yberion** [zitadel/zitadel#7788] [Q&A] [answered]
Hello,

Is there a way to set given name or family name as optional?

For example on an online gamer community, this makes little sens to force people to set those fields.

**Comment by hifabienne** [comment, marked as answer]
Hi @Yberion 
It is not yet possible, however we are working on user schemas. which allow you to define how a user should look like.
You can track the status here: https://github.com/zitadel/zitadel/issues/6433

**Reply by Yberion** [reply]
Awesome, the user schemas will be really powerful, can't wait to get it!
